drivers/net/ethernet/qualcomm/qca_spi.h

```diff
@@ -41,6 +41,7 @@
 
 #define QCASPI_TX_RING_MAX_LEN 10
 #define QCASPI_TX_RING_MIN_LEN 2
+#define QCASPI_RX_MAX_FRAMES 4
```

Comment:
maybe I am mixing stuff, but regarding the naming, is this related to RX ring paramters? And is this related to this commit https://github.com/chargebyte/linux/pull/3/commits/65a92c8162f9451d9a995d5b880be8ac9632b6e5?

Reply:
The problem is there is no such a thing as RX ring. Yes, this value is the same thing, but this is the reason it has been dropped from ethtool -G support.